saleor/graphql/schema.graphql

```diff
@@ -222,6 +222,7 @@ type App implements Node & ObjectWithMetadata {
   appUrl: String
   version: String
   accessToken: String
+  extensions: [AppExtension!]
```

Comment:
Is this field nullable or will it return an empty list when there are no extensions?